Remove seed generator bias

u-funciton opened the pull request:
The old code discards hex digits that are 0, and therefore biases the seed towards more 1 bits. This resolves this by making sure every byte is encoded to two hex digits.

u-funciton committed:
Unbiased seed generator.

The old code discards 0 hex digits, and therefore biases the seed towards more 1 bits. 
This resolves this by making sure every byte is encoded to two hex digits.

mkej commented on app/src/main/java/co/nano/nanowallet/NanoUtil.java:
```diff
@@ -25,11 +25,14 @@
+      
+        StringBuilder sb = new StringBuilder(numchars);
+        for(byte b: randomBytes){        
+            sb.append(String.format("%02x", b));
```
Use `"%02X"` (upper case X) instead and you won't have to convert resulting string to upper case.

u-funciton replied:
You're right, fixed!

u-funciton committed:
Format using "%02X" in generateSeed 

As per mkej's suggestion, formatting with "%02X" eliminates the need of toUpperCase()
